webrtc.html

```
@@ -3153,6 +3155,25 @@ <h2>Methods</h2>
                     </ol>
                   </li>
                 </ol>
+                <p>
+                  A candidate is <dfn>administratively prohibited</dfn> if the
+                  UA has decided not to allow connection attempts to this
+                  address.
+                  The UA MUST prohibit connections to addresses on the
+                  [[!Fetch]] [= block bad port =] list, and MAY choose
+                  to not make connections to other addresses.
+                </p>
+                <p>
+                  If the {{RTCConfiguration/iceTransportPolicy}} member of the {{RTCConfiguration}} is {{RTCIceTransportPolicy/relay}}, candidates requiring external
+                  resolution, such as mDNS candidates and DNS candidates,
+                  MUST be prohibited.
+                </p>
```

Consider rejecting these candidates rather than silently dropping them.